Response/index.ts

```diff
@@ -49,14 +65,16 @@ export namespace Response {
 						typeof response == "object" && !Array.isArray(response)
 							? (({ header, ...body }) => body)(response)
 							: response,
+					...((response.webSocket && { socket: new Factory(response.webSocket) }) ||
+						(response.socket && { socket: response.socket })),
 			  }
 		if (!result.header.contentType)
 			switch (typeof result.body) {
 				case "undefined":
 					break
 				default:
 				case "object":
-					result.header.contentType = contentType ?? "application/json; charset=utf-8"
+					result.header.contentType = typeof contentType == "string" ? contentType : "application/json; charset=utf-8"
```

Comment:
How can `contentType` not be a string or undefined?

Reply:
it could be a ResponseHeader. updated to handle that case as well.